docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -267,14 +256,16 @@ novadb_svc_records=$(oc exec openstack-cell1-galera-0 -c galera -- mysql -rs -ur
 diff -Z <(echo $novadb_svc_records) <(echo $PULL_OPENSTACK_CONFIGURATION_NOVA_COMPUTE_HOSTNAMES)
 ----
 
-. During the pre/post checks the pod `mariadb-client` might have returned a pod security warning
-related to the `restricted:latest` security context constraint. This is due to default security
-context constraints and will not prevent pod creation by the admission controller. You'll see a
-warning for the short-lived pod but it will not interfere with functionality.
-. Delete the `mariadb-data` pod and `mariadb-copy-data` persistent volume claim with databases backup (consider making a snapshot of it, before deleting)
+. Delete the `mariadb-data` pod and `mariadb-copy-data` persistent volume claim with databases backup:
```

Comment:
Is "databases backup" a tool or a command or something that deletes the pod and PVC? If possible, maybe specify what "databases backup" is to clarify the connection between it and the deletions.

Reply:
I don't think "databases backup" is a tool. 

@bogdando Is this step saying that users should consider backing up their databases, then delete the mariadb-data pod and mariadb-copy-data? We need to clarify the connection between the databases backup and the deletions.

Reply:
the volume claim containing the database backup